src/uu/users/src/users.rs

```diff
@@ -10,59 +10,44 @@
 // Allow dead code here in order to keep all fields, constants here, for consistency.
 #![allow(dead_code)]
 
-extern crate getopts;
+extern crate clap;
+#[macro_use]
 extern crate uucore;
 
 use uucore::utmpx::*;
 
-use getopts::Options;
+use clap::{App, Arg};
 
-static NAME: &str = "users";
+static ABOUT: &str = "Output who is currently logged in according to FILE.";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
-pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
-
-    let mut opts = Options::new();
-
-    opts.optflag("h", "help", "display this help and exit");
-    opts.optflag("V", "version", "output version information and exit");
-
-    let matches = match opts.parse(&args[1..]) {
-        Ok(m) => m,
-        Err(f) => panic!("{}", f),
-    };
-
-    if matches.opt_present("help") {
-        println!("{} {}", NAME, VERSION);
-        println!();
-        println!("Usage:");
-        println!("  {} [OPTION]... [FILE]", NAME);
-        println!();
-        println!(
-            "{}",
-            opts.usage("Output who is currently logged in according to FILE.")
-        );
-        return 0;
-    }
+static ARG_FILES: &str = "files";
 
-    if matches.opt_present("version") {
-        println!("{} {}", NAME, VERSION);
-        return 0;
-    }
+fn get_usage() -> String {
+    format!("{0} [FILE]...", executable!())
```

Comment:
```suggestion
    format!("{0} [FILE]", executable!())
```
Only one is allowed.